.buildkite/scripts/steps/functional/osquery_cypress.sh

```diff
@@ -4,8 +4,9 @@ set -euo pipefail
 
 source .buildkite/scripts/common/util.sh
 
+export BUILD_TS_REFS_DISABLE=false
 .buildkite/scripts/bootstrap.sh
-.buildkite/scripts/download_build_artifacts.sh
```

Comment:
Why did you change this? This should download the build artifacts so `build_kibana_platform_plugins` shouldn't be run on every task.

Reply:
Hey, I think this has been resolved in the previous PR :)